### DISCUSSION TITLE
iOS client application: review process with App Store need Oauth Client

### DISCUSSION THREAD

**Discussion by victorbalssa** [firefly-iii/firefly-iii#6127] [General]
Hello @JC5,

I'm currently creating an open source client for iOS (https://github.com/victorbalssa/abacus) and at the last step of the App Store validation.
I was wondering if we can add a Oauth2 client by default in the https://demo.firefly-iii.org application?

This way App Store validator will be able to try the demo version at any time.

![image](https://user-images.githubusercontent.com/12813321/171739731-75b18621-9920-4187-9237-14ed70752539.png)

Name: `ios`
Redirection ~~URL~~ **URI**: `abacusiosapp://redirect`
Confidential: *leave this blank*

Thank you so much for this amazing app! 💪🏻 
Vic

**Comment by JC5** [comment]
I'm sorry, I can't do that for your app. You'll have to instruct them to generate a client themselves (on the demo site) or host a demo site yourself.

**Reply by victorbalssa** [reply]
Ok thank you for your quick answer.
I'll host a demo site.

**Reply by joseluislucio** [reply]
Amazing app for ios ... keep pushing
